plugin/etcd/README.md

```diff
@@ -45,6 +46,7 @@ etcd [ZONES...] {
   queries for those zones will be subject to fallthrough.
 * **PATH** the path inside etcd. Defaults to "/skydns".
 * **ENDPOINT** the etcd endpoints. Defaults to "http://localhost:2379".
+* `credentials` If the etcd cluster pointed by `endpoint` is password protectd, use this field to specify the username and password.
```

Comment:
protectd -> protected

better will be:
~~~
`credentials` is used to set the **USERNAME** and **PASSWORD** for accessing the etcd cluster.
~~~